ui/app/components/clients/page/token.hbs

```diff
@@ -3,199 +3,99 @@
-             or viewing the current month filtered down to auth method }}
-            <Clients::UsageStats
-              @title="Total usage"
-              @description="These totals are within
-              {{if this.selectedAuthMethod this.selectedAuthMethod 'this namespace and all its children'}}.
-              {{if
-                this.isCurrentMonth
-                "Only totals are available when viewing the current month. To see a breakdown of new and total clients for this month, select the 'Current Billing Period' filter."
-              }}"
-              @totalUsageCounts={{this.totalUsageCounts}}
-            />
-          {{/unless}}
-          {{#if this.byMonthActivityData}}
-            <Clients::Token::MonthlyNew
-              @byMonthActivityData={{this.byMonthActivityData}}
-              @authMount={{this.selectedAuthMethod}}
-              @runningTotals={{this.totalUsageCounts}}
-              @responseTimestamp={{this.responseTimestamp}}
-            />
-          {{/if}}
-          {{#if this.hasAttributionData}}
-            <Clients::Attribution
-              @totalUsageCounts={{this.totalUsageCounts}}
-              @newUsageCounts={{this.newClientCounts}}
-              @totalClientAttribution={{this.totalClientAttribution}}
-              @newClientAttribution={{this.newClientAttribution}}
-              @selectedNamespace={{this.selectedNamespace}}
-              @startTimestamp={{this.startMonthTimestamp}}
-              @endTimestamp={{this.endMonthTimestamp}}
-              @responseTimestamp={{this.responseTimestamp}}
-              @isHistoricalMonth={{and (not this.isCurrentMonth) (not this.isDateRange)}}
-              @upgradeExplanation={{this.upgradeExplanation}}
-            />
-          {{/if}}
-          {{#if this.hasMultipleMonthsData}}
-            <Clients::MonthlyUsage
-              @verticalBarChartData={{this.byMonthActivityData}}
-              @responseTimestamp={{this.responseTimestamp}}
-            />
-          {{/if}}
-        {{/if}}
-      {{/if}}
-    {{else if (and (not @model.licenseStartTimestamp) (not this.startMonthTimestamp))}}
-      {{! Empty state for no billing/license start date }}
-      <EmptyState @title={{this.versionText.title}} @message={{this.versionText.message}} />
-    {{else}}
-      <EmptyState
-        @title="No data received  {{if this.dateRangeMessage this.dateRangeMessage}}"
-        @message="Select a different start date above, or choose a different end date here:"
-      >
-        <DateDropdown
-          @handleSubmit={{this.handleClientActivityQuery}}
-          @validateDate={{this.isEndBeforeStart}}
-          @dateType="endDate"
-          @submitText="View"
-        />
-      </EmptyState>
-    {{/if}}
-  {{/if}}
-</div>
+      <StatText
+        @label="Average new clients per month"
+        @value={{this.average this.byMonthNewClients "clients"}}
```

Comment:
Same with this number, it will include sync clients so I think we should just remove or sum new entity+non-entity clients client side